build.cmd

```diff
@@ -0,0 +1,2 @@
+@gulp _dev-build
```

Comment:
I don't see why we should have a script that just invokes a single gulp task on windows, let's not check it in..

Reply:
Are you okay with me adding build.cmd to .gitignore, so I can keep it on my folder?

Reply:
You can put it in `.git/info/exclude` which is like a private gitignore

Reply:
I didn't know that was a thing... Thanks! :)